tools/ratbag-command.c

```diff
@@ -101,6 +101,8 @@ usage(void)
 	       "Profile Commands:\n"
 	       "  profile active get		Print the currently active profile\n"
 	       "  profile active set N		Set profile N as to the  active profile\n"
+	       "  profile enabled get           Check if profile is enabled\n"
+	       "  profile enabled set E         Set profile enabled state to E\n"
```

Comment:
it's worth noting here: ratbag-command is a **debugging tool** for us, not a proper commandline interface that should be used by others. So we have some leeway here, e.g. "profile enabled get" could return multiple values.
I'm open to other ideas though.